src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json

```diff
@@ -720,6 +840,96 @@
           },
           "AliasPart": { "Alias": "main-menu" }
         },
+        {
+          "ContentItemId": "[js: variables('tagsContentItemId')]",
+          "ContentType": "Taxonomy",
+          "DisplayText": "Tags",
+          "Latest": true,
+          "Published": true,
+          "TitlePart": {
+            "Title": "Tags"
+          },
+          "AliasPart": {
+            "Alias": "tags"
+          },
+          "TaxonomyPart": {
+            "Terms": [
+              {
+                "ContentItemId": "[js: variables('earthTagContentItemId')]",
+                "ContentType": "Tag",
+                "DisplayText": "Earth",
+                "Tag": {},
+                "TitlePart": {
+                  "Title": "Earth"
+                }
+              },
+              {
+                "ContentItemId": "[js: variables('explorationTagContentItemId')]",
+                "ContentType": "Tag",
+                "DisplayText": "Exploration",
+                "Tag": {},
+                "TitlePart": {
+                  "Title": "Exploration"
+                }
+              },
+              {
+                "ContentItemId": "[js: variables('spaceTagContentItemId')]",
+                "ContentType": "Tag",
+                "DisplayText": "Space",
+                "Tag": {},
```

Comment:
What is this property ?

Reply:
the `Tag{}` property? 
I imagine it's the Fields property for the tags content item. Which content type is called `Tags` of course. 
So the deployment plan exported an empty property to hold the fields, that it doesn't actually have (unless you add some fields).
I can remove it, if you prefer. 
It won't break anything. but that's what a deployment plan does.